File: onlinejudge/programmers/level_1/서울에서 김서방 찾기.java
class Solution {
    private final static String KIM = "Kim";
    public String solution(String[] seoul) {
        String index = "";
        for (int i = 0; i < seoul.length; i += 1){
            if (seoul[i].equals(KIM)) {
                index = String.valueOf(i);
                break;
            }
        }
        return "김서방은 " + index + "에 있다";
    }
}

File: onlinejudge/programmers/1/서울에서 김서방 찾기.go
package level_1

import (
	"strconv"
)

func solution(seoul []string) string {
	var a int = 0
	for i := 1; i < len(seoul); i += 1 {
		if seoul[i] == "Kim" {
			a = i
		}
	}
	return "김서방은 " + strconv.Itoa(a) + "에 있다"
}
